docs/disciplines/Imaging/index.md

```diff
@@ -1,4 +1,95 @@
+# Medical Image Formats
 
-# Introduction
+## Introduction
 
-TODO:: Add a short description here
+A medical image is a digital representation of the internal structure or
+function of an anatomic region, typically presented as an array of picture
+elements called **pixels** (2D) or **voxels** (3D). This representation is a
+discrete mapping of **numerical values** to positions in space.
+
+<!-- - The numerical values, known as **intensity values**, vary based on:
+    - **Image type**: MRI, CT, PET, etc.
+    - **Acquisition method**
+    - **Reconstruction and post-processing**
+
+*add diagram showing pixel and voxel arrangement* -->
+
+## Medical Image Metadata
+
+Medical images often come with **metadata**, which provides additional
+information about the image. This metadata is usually stored at the beginning of
+the image file as a "header."
+
+### Common Metadata Fields
+
+- **Image dimensions**: Width, height, depth
+- **Voxel size**: Spacing between voxels
+- **Origin**: Location of the first voxel
+- **Orientation**: Direction of x, y, and z axes
+- **Pixel depth**: Bytes used to represent each voxel intensity
+- **Data type**: Integer, floating-point, etc.
+
+*add diagram showing a sample header with key metadata fields*
+
+## Pixel Data
+
+The **pixel data** in a medical image file represents the actual image values,
+stored in a format specific to the image file.
+
+- In fixed-size header formats, pixel data follows the header directly.
+- In other formats, a **marker** or **tag** indicates the start of pixel data.
+
+### **Pixel Data Size**
+
+\[
+\text{Pixel Data Size} = \text{Rows} \times \text{Columns} \times
+\text{Pixel Depth (Bytes)} \times \text{Number of Frames}
+\]
+
+### **Image File Size**
+
+\[
+\text{Image File Size} = \text{Header Size} + \text{Pixel Data Size}
+\]
```

Comment:
not sure if it's a markdown formatting issue, but it isn't clear what you're speaking about here or what this refers to